heartbeat/pgsql

```diff
@@ -1915,14 +1946,44 @@ pgsql_validate_all() {
         else
             CHECK_XLOG_LOC_SQL="select pg_last_xlog_replay_location(),pg_last_xlog_receive_location()"
         fi
-        CHECK_REPLICATION_STATE_SQL="select application_name,upper(state),upper(sync_state) from pg_stat_replication"
+
+        NODE_LIST=`echo $OCF_RESKEY_node_list | tr '[A-Z]' '[a-z]'`
+
+        if [ -n "$OCF_RESKEY_replication_multiple_slaves" ];then
```

Comment:
You could do a `if ocf_is_true "$OCF_RESKEY_replication_multiple_slaves"; then` instead, and set it to e.g. no or off in the _default on top for clarity.

Reply:
This is a nice one. Thanks! Fixed it with ocf_is_true which is definitely more readable.